.build-win/view-object-builder.deps.json

```diff
@@ -1199,14 +1199,14 @@
         },
         "native": {
           "runtimes/win-x64/native/hostfxr.dll": {
-            "fileVersion": "2.0.26615.1"
+            "fileVersion": "0.0.0.0"
           }
         }
       },
       "runtime.win-x64.runtime.native.System.Data.SqlClient.sni/4.4.0": {
         "native": {
           "runtimes/win-x64/native/sni.dll": {
-            "fileVersion": "4.6.25512.1"
+            "fileVersion": "0.0.0.0"
```

Comment:
All of these fileVersion changes are a little concerning, but I have no idea why they happened or if they're an issue 🤷‍♂️

Reply:
Is there a good workflow that uses this we could test it out in?  Just so we can verify this didn't break anything?